.changeset/tricky-toys-teach.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': patch
+---
+
+Remove deprecated `create-plugin` script
```

Comment:
This might actually not be clear enough. A user could interpret that the `create-plugin` is removed from the CLI but it isn't. Maybe just a line that the script command is removed from newly generated Backstage apps should be enough 😄

Reply:
I will come back to this once the scope of this PR is determined